app/views/submissions/_missing_assignment_file_toggle_list.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= javascript_include_tag "jquery-ui" %>
```

Comment:
This shouldn't be necessary. Generally you shouldn't include JS files in a partial. But also the only use of jQuery in this file is on line 4 (and this uses `jquery` not `jquery-ui`) but jQuery should already be included in this template.